### PR TITLE
fix: recount coordinates with scroll px for ignore element in screens…

### PR DESCRIPTION
**Pull request by patrjam**
…hotElement

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -630,6 +634,10 @@ class ResembleHelper extends Helper {
       };
     });
 
+    if (scrollOffset.X || scrollOffset.Y){
+    this.debug(`Screenshotted element was in test scrolled "${JSON.stringify(scrollOffset.Y)}" px vertically and "${JSON.stringify(scrollOffset.X)}" px horizontal.`);
```
Comment (Trawais) [inline index.ts:638]: missing spaces for correct indent

File: index.ts [hunk 2]
```diff
@@ -630,6 +634,10 @@ class ResembleHelper extends Helper {
       };
     });
 
+    if (scrollOffset.X || scrollOffset.Y){
```
Comment (Trawais) [inline index.ts:637]: missing space before `{` ?

File: index.ts [hunk 3]
```diff
@@ -614,13 +614,17 @@ class ResembleHelper extends Helper {
    * @returns {Promise<{ignoredBoxes: [{left: *, top: *, right: *, bottom: *}]>}
    */
   private async _reCountElementCoordinatesForIgnoreInScreenshotElement(selector: Selector, ignoredElementsCoordinates: BoxCoordinates[]): Promise<BoxCoordinates[]> {
+    const helper = this._getHelper();
     const boundingBox = await this._getBoundingBox(selector);
 
+    const scrollOffset = await helper.executeScript(() => ({ X: window.pageXOffset, Y: window.pageYOffset }));
```
Comment (Trawais) [inline index.ts:620]: Did you test it for both WebDriver and Playwright?

**Review by Trawais — CHANGES_REQUESTED** [review]

File: index.ts (hunk 1 above)
Comment (patrjam) [reply index.ts]: Resolved.

File: index.ts (hunk 2 above)
Comment (patrjam) [reply index.ts]: Resolved.

File: index.ts (hunk 3 above)
Comment (patrjam) [reply index.ts:620]: executeScript is a standard way for both drivers/helpers, this was tested in the past and worked good.

**Review by Stanin84 — APPROVED** [review]

**Review by Trawais — APPROVED** [review]

**Pull request merged**